dev/sg/internal/docker/docker.go

```diff
@@ -0,0 +1,48 @@
+package docker
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/cockroachdb/errors"
+	"github.com/docker/cli/cli/config/configfile"
+	"github.com/docker/docker-credential-helpers/client"
+	"github.com/docker/docker-credential-helpers/credentials"
+)
+
+func getStoreProvider() (string, error) {
+	// TODO:
+	// This is a hack to get the store provider.
+	// It may not work with Windows.
+	data, err := ioutil.ReadFile(fmt.Sprintf("%s/.docker/config.json", os.Getenv("HOME")))
```

Comment:
```suggestion
	data, err := ioutil.ReadFile(fmt.Sprintf("%s/.docker/config.json", os.UserHomeDir()))
```

https://pkg.go.dev/os#UserHomeDir

Reply:
😂 copilot failed me